corporate/models.py

```diff
@@ -17,7 +20,10 @@ class Customer(models.Model):
     and the active plan, if any.
     """
 
-    realm: Realm = models.OneToOneField(Realm, on_delete=CASCADE)
+    realm: Optional[Realm] = models.OneToOneField(Realm, on_delete=CASCADE, null=True, blank=True)
+    remote_server: Optional[RemoteZulipServer] = models.OneToOneField(
+        RemoteZulipServer, on_delete=CASCADE, null=True, blank=True
+    )
```

Comment:
I'm guess `blank=True` might be technically correct; we don't really use the Django forms system, so we don't generally make a habit of setting that property (which is only used for said system).